mailchimp3/mailchimpclient.py

```diff
@@ -46,9 +46,15 @@ def __init__(self, mc_user, mc_secret, enabled=True):
         """
         super(MailChimpClient, self).__init__()
         self.enabled = enabled
-        self.auth = HTTPBasicAuth(mc_user, mc_secret)
-        datacenter = mc_secret.split('-').pop()
-        self.base_url = 'https://{}.api.mailchimp.com/3.0/'.format(datacenter)
+        if access_token:
```

Comment:
That sounds good. Add a link to the instructions in the README files and just place the information on initialization for tokens right after the credential init, with the explanation that you need to get a token from wherever. Also, please bump the version numbers in the README and setup files (I believe the README files just have images that generate based on the name which includes version number)